uitest/src/test/java/com/vaadin/tests/components/combobox/ComboBoxSelectingNewItemValueChangeTest.java

```diff
@@ -153,6 +153,8 @@ private void typeInputAndSelect(String input, SelectionType selectionType) {
     }
 
     private void sendKeysToInput(CharSequence... keys) {
+        // ensure mouse is located over the ComboBox to avoid hover issues
+        new Actions(getDriver()).moveToElement(comboBoxElement).perform();
```

Comment:
Is the functionality broken when the mouse is not hovering the ComboBox or is the issue in the mouse hovering over an entry in the list ?

Reply:
Issue is mouse hovering over an entity in the list.